apps/webClient/src/views/Home/index.tsx

```diff
@@ -56,47 +56,21 @@ const Home: React.FC<Props> = ({}) => {
                   sollicitudin aliquam ultrices sagittis.
                 </i>
               </Typography>
+              <Box mt={2}>
+                <Grid container justify="center">
+                  <FlareIcon fontSize="large" />
+                </Grid>
+              </Box>
             </Container>
           </Grid>
           <Grid item container sm={3} justify="flex-end">
-            <NavButton route="about" text="about us" />
+            <NavButton route="about" text="about us" vertical={true} />
           </Grid>
         </Grid>
       </Box>
-      <Box mt={12}>
-        <Grid container justify="space-around">
-          <Container maxWidth="xs">
-            <Typography
-              variant="h5"
-              align="center"
-              className={classes.contrastHeader}
-              gutterBottom
-            >
-              partner name
-            </Typography>
-            <Typography variant="body1" align="center" color="textSecondary">
-              <i>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                sitame etel
-              </i>
-            </Typography>
-          </Container>
-          <Container maxWidth="xs">
-            <Typography
-              variant="h5"
-              align="center"
-              className={classes.contrastHeader}
-              gutterBottom
-            >
-              partner name
-            </Typography>
-            <Typography variant="body1" align="center" color="textSecondary">
-              <i>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                sitame etel
-              </i>
-            </Typography>
-          </Container>
+      <Box mt={6}>
+        <Grid container justify="center">
+          <NavButton route="lets-talk" text="lets talk" vertical={false} />
```

Comment:
Not sure how to make props optional in typescript, or set a default if no prop is passed, so that I don't have to pass this prop unless I want it vertical. I think this is fine thought.

Reply:
Google is your friend `typescript optional interface property`

```
interface Person {
    name: string;
    age: number;
    phone?: string;
}
```

Typescript has nothing to do with actually setting a default, that's a javascript thing

({ number = 5 }) => {